e2e/bitrise.yml

```diff
@@ -0,0 +1,62 @@
+format_version: "17"
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+
+workflows:
+  test_list_tests:
+    envs:
+    - TEST_APP_URL: https://github.com/bitrise-io/sample-swift-project-with-parallel-ui-test.git
+    - TEST_APP_BRANCH: master
+    - BITRISE_PROJECT_PATH: ./_tmp/BullsEye.xcworkspace
+    - BITRISE_SCHEME: BullsEye
+    steps:
+    - script:
+        inputs:
+        - content: |-
+            #!/bin/env bash
+            set -xeo pipefail
+            rm -rf "./_tmp"
+    - git::https://github.com/bitrise-steplib/bitrise-step-simple-git-clone.git:
+        inputs:
+        - repository_url: $TEST_APP_URL
+        - branch: $TEST_APP_BRANCH
+        - clone_into_dir: ./_tmp
+    - xcode-build-for-test:
+        inputs:
+        - project_path: $BITRISE_PROJECT_PATH
+        - scheme: $BITRISE_SCHEME
+        - xcodebuild_options: -testPlan FullTests
+        - destination: generic/platform=iOS Simulator
+    - path::./:
+        inputs:
+        - product_path: $BITRISE_XCTESTRUN_FILE_PATH
+        - shard_count: 3
+    - script:
```

Comment:
Note: there is a helper step (`git::https://github.com/bitrise-steplib/bitrise-step-check-step-outputs.git@main`) that may be used: https://github.com/bitrise-steplib/steps-xcode-test/blob/2d52e4b24129b1f4d36d85eab21266fd4da2e8db/e2e/bitrise.yml#L185

Reply:
We could also check the contents of the outputted shards